Lib/test/test_regrtest.py

```diff
@@ -663,7 +663,7 @@ def list_regex(line_format, tests):
     def parse_random_seed(self, output):
         match = self.regex_search(r'Using random seed ([0-9]+)', output)
         randseed = int(match.group(1))
-        self.assertTrue(0 <= randseed <= 100_000_000, randseed)
```

Comment:
Yeah, I don't think that we should check the expected entropy here. We just have to check that the message is logged.